browse-scaler/src/cmr.js

```diff
@@ -21,6 +17,8 @@ exports.getBrowseImageFromConcept = async concept => {
     const imgRegex = /\b(.png|.jpg|.gif|.jpeg)$/;
     const imgurl = links.filter(link => imgRegex.test(link.href))[0];
 
+    console.log(`links from metadata ${JSON.stringify(links)}`);
```

Comment:
Is there a concept of log levels that could be used?

Reply:
Looks like we could use this library: https://www.npmjs.com/package/winston